.changeset/stupid-keys-brush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-documentation': minor
+---
+
+Added getting-started docs page for components-angular package. Updated the getting-started page of the components package.
```

Comment:
The info that a page was added is useful here. As for the second line, @oliverschuerch, please specify what was changed with the update so it's clear from the changelog and people don't have to go search and diff the docs. This could also be a separate changeset.